src/star-rating/index.js

```diff
@@ -0,0 +1,33 @@
+const simpleIcons = require("simple-icons");
+const starSVG = require("./star.svg.js");
+const starRatingComponentStyles = require("./index.style");
+
+const starRating = async ({ skill, text }) => {
+  let content = "";
+  Array.from({ length: 5 }).map((_r, index) => {
+    if (index < parseInt(text)) {
+      content += starSVG();
```

Comment:
`#FFD700` maybe we can send colors from here and take colors if present or else we can fallback with none

Reply:
Yes, for fill we can have `none` as default fallback. What should be the default fallback stroke color? The same `#FFD700` ?